src/State/Scope.php

```diff
@@ -98,6 +104,31 @@ public function setTags(array $tags): self
         return $this;
     }
 
+    /**
+     * Sets context data with the given name.
+     *
+     * @param string               $name  The name that uniquely identifies the context
+     * @param array<string, mixed> $value The value
```

Comment:
Shouldn't this be `array<string, array<string, mixed>>`?

Reply:
I will try another time but I did have to revert this because psalm says it looks like an object